package.json

```diff
@@ -6,7 +6,7 @@
   "devDependencies": {
     "@cyclonedx/bom": "^3.8.0",
     "@types/fs-extra": "9.0.13",
-    "@types/jest": "27.0.2",
+    "@types/jest": "29.5.8",
```

Comment:
Latest `@types/jest` is `29.6.3` and used jest is `29.7.0` - Could we bump this to at least `29.6.3`?

Reply:
According to [npm](https://www.npmjs.com/package/@types/jest) the latest is 29.5.8, that's why I use this one.

[GH](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/657a7864cd96fc3159f8c47ac6a1d4d5330a9742/types/jest/package.json#L4C1-L5C1) shows 29.5.999, but where do you see `29.6.3`/

Reply:
My bad, was confused with another one 👍